rtdata/languages/Francais

```diff
@@ -663,7 +663,7 @@ HISTORY_MSG_421;Retinex - Gamma
 HISTORY_MSG_422;Retinex - Gamma
 HISTORY_MSG_423;Retinex - Pente Gamma
 HISTORY_MSG_424;Retinex - Seuille HL
-HISTORY_MSG_425;Retinex - Base Log
+HISTORY_MSG_425;Retrait de taches
```

Comment:
There is a problem here. The history message should not/cannot be reused afaik. Interestingly, there is no similar change to the `default` language file.

Reply:
This is an artifact of merging history messages from dev. Going forward, we should stop using numbered history messages.